src/interpreter/code.h

```diff
@@ -133,10 +136,19 @@ namespace verona::interpreter
       return header;
     }
 
+    void parse_verona_nums(size_t& ip) {
```

Comment:
This doesn't really parse anything.
How about `check_magic_number` instead? Also this should be a private method.